code/modules/reagents/oldchem/reagents/reagents_med.dm

```diff
@@ -110,11 +110,11 @@
 /datum/reagent/cryoxadone/on_mob_life(var/mob/living/M as mob)
 	if(!M) M = holder.my_atom
 	if(M.bodytemperature < 265)
-		M.adjustCloneLoss(-4)
```

Comment:
It's harder to get, but arguably no side effects

>/datum/reagent/rezadone/on_mob_life(mob/living/M)
	M.setCloneLoss(0) //Rezadone is almost never used in favor of cryoxadone. Hopefully this will change that.
	M.adjustCloneLoss(-1) //What? We just set cloneloss to 0. Why? Simple; this is so external organs properly unmutate.
	M.adjustBruteLoss(-1)
	M.adjustFireLoss(-1)
	M.status_flags &= ~DISFIGURED
	..()
	return

It has an OD though, but who needs that when it literally changes your cloneloss to 0